pkg/cni/cni_test.go

```diff
@@ -27,6 +27,10 @@ import (
 	"github.com/containernetworking/cni/pkg/ns"
 )
 
+const (
+	dirMode = os.FileMode(0750)
```

Comment:
`os.FileMode(0750)` is `-rwxr-x---`. I think we want `drwxr-x---`, and thus that should be:

```Go
      dirMode = os.FileMode(0750) | os.ModeDir